bench/src/template.cpp

```diff
@@ -0,0 +1,27 @@
+#include "benchmark/benchmark_api.h"
+#include "benchmark/benchmark.h"
+
+
+using namespace Tangram;
+
+class TemplateFixture : public benchmark::Fixture {
+public:
+    // std::unique_ptr<Template> ctx;
+
+    void SetUp() override {
+        ctx = std::make_unique<TestContext>();
```

Comment:
nit; 
This is a template and is not compiled still make sense to have it clean. Just comment out lines using `ctx`.